OpenRA.Mods.Common/Widgets/Logic/GameSaveBrowserLogic.cs

```diff
@@ -198,7 +198,11 @@ void LoadGames(ScrollItemWidget gameTemplate, ScrollItemWidget newTemplate)
 				item.IsVisible = () => true;
 				item.IsSelected = () => selectedSave == item.ItemKey;
 				item.OnClick = () => Select(item.ItemKey);
-				item.OnDoubleClick = Load;
+
+				if (isSavePanel)
+					item.OnDoubleClick = () => Save(world);
```

Comment:
What do you think about assigning `world` to an instance variable and removing it from the `Save` signature?
This line could then be written as `item.OnDoubleClick = isSavepanel ? Save : Load;`.